inspire_json_merger/config.py

```diff
@@ -74,6 +78,7 @@ class ArxivOnArxivOperations(MergerConfigurationOperations):
         'preprint_date',
         'public_notes',
         'publication_info',
+        _ORDER_KEY_TO_FILTER_OUT,
```

Comment:
I don't think any of the changes to this file are needed now that you only add positioning info to head, as it won't ever conflict.

Reply:
:+1:

Reply:
you didn't change this, I don't think you can have conflicts now, can you?